packages/jest-config/jest.config.mjs

```diff
@@ -0,0 +1,15 @@
+/**
+ * @type {import('jest').Config}
+ * */
+export default {
+  coverageProvider: 'v8',
+  coverageThreshold: {
+    global: {
+      branches: 100,
+      functions: 100,
+      lines: 100,
+      statements: 100,
+    },
+  },
+  testEnvironment: 'jest-environment-jsdom',
```

Comment:
We zouden deze regel aan het project zelf over kunnen laten voor als je geen DOM related dingen zou willen testen, maar ik denk niet dat het in de weg zit.